src/gui/MatchTemplatePanel.cpp

```diff
@@ -465,6 +465,22 @@ void MatchTemplatePanel::SetInputsForPossibleReRun(bool set_up_to_resume_job) {
         ResumeRunCheckBox->SetValue(true);
         ResumeRunCheckBox->Enable(true);
         ResetAllDefaultsButton->Enable(false);
+
+        ReferenceSelectPanel->SetSelection(volume_asset_panel->ReturnArrayPositionFromAssetID(job_to_resume->ref_volume_asset_id));
```

Comment:
Often, ppl want the (why) in comments, but this is convoluted enough that even just the (what) would be very useful.